packages/server/logging/operationConstants.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Helper constants for operation status when logging.
+ * Intended to be used with the `operationStatus` field when logging.
+ * Helps to avoid typos and ensure consistency.
+ */
+export const STATUS = {
```

Comment:
Not a fan of the server/logging folder that has no organization and has various helpers, utils and constants all stored basically at the same level. Ideally it should follow a structure similar to speckle modules under /modules